pandas/core/strings.py

```diff
@@ -1314,21 +1314,42 @@ def str_index(arr, sub, start=0, end=None, side='left'):
 
 def str_pad(arr, width, side='left', fillchar=' '):
     """
-    Pad strings in the Series/Index with an additional character to
-    specified side.
+    Pad strings in the Series/Index with additional characters on
+    specified side to fill up to specified width.
```

Comment:
Since you're trying to improve the docstring, can we try to make this initial description a single line (per convention)?  If you need to provide more details, you can do so afterwards i.e.

~~~python
<one-line summary>

<description>
~~~

Reply:
Thanks. I'll keep this in mind and correct it now.